packages/svelte-vscode/src/extension.ts

```diff
@@ -323,6 +324,25 @@ function addDidChangeTextDocumentListener(getLS: () => LanguageClient) {
     });
 }
 
+async function addWhereImportsUsedListener(getLS: () => LanguageClient) {
+    //??
+    setTimeout(() => {
```

Comment:
attempt to hardcode test. I assume this would eventually fire off the context menu?